server/controllers/pages/cart.js

```diff
@@ -0,0 +1,13 @@
+const { getItems } = require('../../database/queries/cart');
+
+const getCart = (req, res) => {
+  const { userId } = req.cookies;
```

Comment:
You need to get userID from request cookies which are JWT, 
So first of all you need to verify your token.
As you need to check user identity in multiple routes, try to think about creating middleware that can handle this for you.